bindings/gumjs/gumdukinstruction.c

```diff
@@ -21,6 +21,13 @@
 #elif defined (HAVE_ARM64)
 # define GUM_DEFAULT_CS_ARCH CS_ARCH_ARM64
 # define GUM_DEFAULT_CS_MODE CS_MODE_ARM
+#elif defined (HAVE_MIPS)
+# define GUM_DEFAULT_CS_ARCH CS_ARCH_MIPS
+# if defined (G_LITTLE_ENDIAN)
+#  define GUM_DEFAULT_CS_MODE CS_MODE_LITTLE_ENDIAN | CS_MODE_MIPS32
+# else
+#  define GUM_DEFAULT_CS_MODE CS_MODE_BIG_ENDIAN | CS_MODE_MIPS32
```

Comment:
It would be sweet to flip the ordering here, as `CS_MODE_MIPS32` is the primary thing, and the endian is the secondary (plus, it will line up better visually).